### PR TITLE
Cohort Export UI

### PR DESCRIPTION
**Pull request by BPierrick**
Capture d’écran 2021-03-18 à 18 30 52

### CODE REVIEW

File: src/data/crfParameters.ts [hunk 1]
```diff
@@ -1,4 +1,12 @@
-export const CRF_ATTRIBUTES = [
+export type CrfParameter = {
+  officialName: string
+  customName: string
+  att_type: 'identifying' | 'quasiidentifying' | 'insensitive' | 'sensitive'
```
Comment (simonvadee) [inline src/data/crfParameters.ts]: what does "att" mean ?

File: src/components/RedcapExport/RedcapExport.tsx [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+                className={classes.addButton}
+                variant="outlined"
+                onClick={handleAddButton}
+                startIcon={<AddIcon />}
+              >
+                Ajouter une variable
+              </Button>
+            </div>
+          </Grid>
+          {/* <Grid item>
```
Comment (simonvadee) [inline src/components/RedcapExport/RedcapExport.tsx:152]: why is this commented out? is this the old version ?

**Review by MiskoG — APPROVED** [review]
UI-speaking it's fine for me by now :)

File: src/data/crfParameters.ts (hunk 1 above)
Comment (BPierrick) [reply src/data/crfParameters.ts]: attribute I guess. I kept the naming already in place

Comment (simonvadee) [reply src/data/crfParameters.ts]: ok can you rename it to attr_type pls ?

File: src/components/RedcapExport/RedcapExport.tsx (hunk 2 above)
Comment (BPierrick) [reply src/components/RedcapExport/RedcapExport.tsx:152]: Yes, in the resah version there were explaining texts about Redcap export.
Those texts were also refering to a "Champ de recherche" feature that has been removed from the mock, so i just commented it out

**Review by simonvadee — APPROVED** [review]

**Pull request merged**